cmd/acctestgen/main.go

```diff
@@ -0,0 +1,287 @@
+package main
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+	"os"
+	"strings"
+	"text/template"
+
+	"golang.org/x/text/cases"
+	"golang.org/x/text/language"
+)
+
+func main() {
+	var dryrun = flag.Bool("dryrun", false, "preview list of files to be created")
+	flag.Parse()
+
+	service := flag.Arg(0)
+	service = strings.ToLower(service)
+	err := validate(service)
+	if err != nil {
+		fmt.Printf("\nERROR: %s\n\n", err)
+		os.Exit(1)
+	}
+
+	datasourcePath := fmt.Sprintf("internal/services/%s/data_source_test.go", service)
+	resourcePath := fmt.Sprintf("internal/services/%s/resource_test.go", service)
+	testdataDirPath := fmt.Sprintf("internal/services/%s/testdata/", service)
+	resourceDataPath := fmt.Sprintf("internal/services/%s/testdata/resourcebasicconfig.tf", service)
+	datasourceDataPath := fmt.Sprintf("internal/services/%s/testdata/datasourcebasicconfig.tf", service)
+
+	// print preview of files/directories
+	fmt.Println("To be created:")
+	fmt.Printf("- file: %s\n", datasourcePath)
+	fmt.Printf("- file: %s\n", resourcePath)
+	fmt.Printf("- dir: %s\n", testdataDirPath)
+	fmt.Printf("- file: %s\n", resourceDataPath)
+	fmt.Printf("- file: %s\n", datasourceDataPath)
+	fmt.Println()
```

Comment:
nit: i would go with a multi line string here instead for readability.

```suggestion
	fmt.Println(fmt.Sprintf(`To be created:
- file:       %[1]s
- file:       %[2]s
- directory:  %[3]s
- file:       %[4]s
- file:       %[5]s
`, datasourcePath, resourcePath, testdataDirPath, resourceDataPath, datasourceDataPath)
```